### PR TITLE
Multiselect

### PR DESCRIPTION
**Pull request by EHale64**
added the multiselect field for roles to the edit profile page

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class SessionsController < ApplicationController
   def create
+    # require "pry"; binding.pry
```
Comment (twarbelow) [inline app/controllers/sessions_controller.rb:3]: please remove this line

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 class WelcomeController < ApplicationController
-  def index; end
+  def index
```
Comment (twarbelow) [inline app/controllers/welcome_controller.rb:2]: just for future reference, rubocop will prefer the one line syntax for empty methods

File: spec/features/search/search_index_spec.rb [hunk 3]
```diff
@@ -7,7 +7,7 @@
     allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
 
   end
-  xit 'should show piece information' do
+  it 'should show piece information' do
```
Comment (twarbelow) [inline spec/features/search/search_index_spec.rb:10]: thanks for putting these back in

File: spec/features/user/navbar_spec.rb [hunk 4]
```diff
@@ -29,7 +28,7 @@
     end
 
     it 'on Home page after a user is logged in' do
-      click_link "Home"
+      visit "/"
```
Comment (twarbelow) [inline spec/features/user/navbar_spec.rb:31]: if you are doing the path thing, perhaps this should be `visit root_path`

File: spec/features/user/navbar_spec.rb [hunk 5]
```diff
@@ -50,8 +49,14 @@
       expect(page).to have_link("Log Out")
     end
 
+  end
 
+  describe 'logout' do
```
Comment (twarbelow) [inline spec/features/user/navbar_spec.rb:54]: why is this changed?

File: spec/features/user/user_profile_spec.rb [hunk 6]
```diff
@@ -47,12 +48,14 @@
...
 
     fill_in :username, with: "StringsNThings"
+    select 'Teacher', from: 'Role'
     click_on "Submit Changes"
 
     expect(page).to have_content("Username has already been taken")
     expect(current_path).to eq("/users/#{@user.id}/edit")
 
     fill_in :username, with: "Bruce Banner"
+    select 'Teacher', from: 'Role'
```
Comment (twarbelow) [inline spec/features/user/user_profile_spec.rb:58]: would be interested to make sure that editing a user profile and changing the role successfully updates the user profile with the new role and removes the old role in a test.

File: spec/features/welcome/new_user_registration_spec.rb [hunk 7]
```diff
@@ -13,22 +13,25 @@
...
-    click_link('Login With Google')
-
-    fill_in('Username', with: 'Bruce Banner')
-    fill_in('About me', with: 'Hey all, I am trying to learn to play some classical tunes as a form of stress relief')
-    click_on('Submit Changes')
-
-    expect(User.last.username).to eq('Bruce Banner')
-    expect(User.last.about_me).to eq('Hey all, I am trying to learn to play some classical tunes as a form of stress relief')
-  end
+  # it "can edit a new user from the login" do
```
Comment (twarbelow) [inline spec/features/welcome/new_user_registration_spec.rb:19]: why is this commented out and still in the code?

File: spec/features/welcome/new_user_registration_spec.rb [hunk 8]
```diff
@@ -44,4 +47,18 @@
...
   end
+
+  it "can edit a role" do
+    user = User.create!(username:"Tony Stark" ,uid: "12345678" ,access_token: "token" ,email: "tony@stark.com")
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+
+    visit edit_user_path(user.id)
+
+    expect(page).to have_content('Role')
+    #   As an authenticated user,
```
Comment (twarbelow) [inline spec/features/welcome/new_user_registration_spec.rb:58]: why are there comments here?

File: spec/features/welcome_page/new_user_registration_spec.rb [hunk 9]
```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
```
Comment (twarbelow) [inline spec/features/welcome_page/new_user_registration_spec.rb:1]: this spec is a bit confusing at first glance...it looks like we might be reiterating tests from other spec files?

what are the comments left in here for? will we need them in the future?

**Review by twarbelow — CHANGES_REQUESTED** [review]

File: app/controllers/welcome_controller.rb (hunk 2 above)
Comment (jKwibe) [reply app/controllers/welcome_controller.rb:2]: Sure!
